modules/log-upgrade/6x-uninstall-es-operator.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/cluster-logging-uninstall.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="uninstall-es-operator_{context}"]
+= Uninstalling Elasticsearch
+
+You can unintall Elasticsearch by using the {product-title} web console.
```

Comment:
"unintall" has a typo, it's -> uninstall

It should be mentioned that you can uninstall Elasticsearch is it's not used for other component as "Jaeger, Service Mesh or Kiali"

Reply:
done